rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/10_settings.yml

```diff
@@ -180,10 +180,15 @@ set start_time and end_time:
           settings:
             index:
               mode: time_series
-              routing_path: [metricset]
+              routing_path: [ metricset ]
```

Comment:
I don't *think* this should make a difference, right?

Reply:
ok, maybe I format it, I will revert it